Initial commit - frontier group (#34880)
DataDog/documentation: content/en/security/cloud_security_management/frontier_group.md

---
title: Frontier Group
further_reading:
- link: "/security/cloud_security_management/misconfigurations/"
  tag: "Documentation"
  text: "Start tracking misconfigurations with Cloud Security Misconfigurations"
- link: "/security/cloud_security_management/identity_risks/"
  tag: "Documentation"
  text: "Understand your identity landscape with Cloud Security Identity Risks"
- link: "/security/cloud_security_management/vulnerabilities/"
  tag: "Documentation"
  text: "Learn more about Cloud Security Vulnerabilities"
---

{{< learning-center-callout header="Join the frontier group" hide_image="true" btn_title="Sign Up" btn_url="https://frontier-signup.vercel.app/">}}

>   An opt-in program where you get access to the newest AI features we're developing. We'll only enable what we're comfortable enabling for our own internal security team.

Reviewer: ```suggestion
The frontier group is an opt-in program where you get access to the newest AI features Datadog is developing. We'll only enable what we're comfortable enabling for Datadog's internal security team.
```

{{< /learning-center-callout >}}

## What's included

### Ownership agent

***Status:*** *available for frontier partners*<br>
Automated inference on who owns a particular resource and should be responsible for remediating it.

### 1-click CSPM remediation

***Status:*** *available for frontier partners*<br>
Automated (except human-in-the-loop) remediation on running cloud resources and the IaC definitions of those resources.

### Unified explorer

***Status:*** *available for frontier partners*<br>
Misconfigurations, Identity Risks, and Attack Paths combined in a single explorer.

### CLI vulnerabilities

***Status:*** *available for frontier partners*<br>
CI/CD vulnerability scans, command line integrations for agentic feedback.

### Container public accessibility

***Status:*** *coming soon*<br>
More accurate public access evaluations of containers through EKS and runtime data.

### Critical assets

***Status:*** *coming soon*<br>
Automatic critical asset inference based on observability data, overridable through manual queries.

> ## Further Reading

Reviewer: ```suggestion
## Further reading
```
